app/instance-initializers/browser/ember-href-to.js

```diff
@@ -1,17 +1,29 @@
 import Em from 'ember';
 import HrefTo from 'ember-href-to/href-to';
 
+let hrefToClickHandler;
+function closestLink(el) {
+  if (el.closest) {
+    return el.closest('a');
+  } else {
+    el = el.parentElement;
+    while (el && el.tagName !== 'A') {
+      el = el.parentElement;
+    }
+    return el;
+  }
+}
 export default {
   name: 'ember-href-to',
-  initialize: function(applicationInstance) {
-    let $body = Em.$(document.body);
-    $body.off('click.href-to', 'a');
-    
-    $body.on('click.href-to', 'a', (e) => {
-      let hrefTo = new HrefTo(applicationInstance, e);
-      hrefTo.maybeHandle();
-
-      return true;
-    });
+  initialize(applicationInstance) {
+    document.body.removeEventListener('click', hrefToClickHandler);
```

Comment:
The first time that this executes, `hrefToClickHandler` will be undefined. If that's intended, perhaps we should add an `if` guard around the `document.body.removeEventListener`? If not, perhaps move the `document.body.removeEventListener` below so that `hrefToClickHandler` has been assigned